bcipy/helpers/task.py

```diff
@@ -370,186 +373,153 @@ def get_key_press(
     return None
 
 
+def data_reshaper(input_data_type: InputDataType, *args, **kwargs):
+    if input_data_type == InputDataType.TRIAL:
+        return trial_reshaper(*args, **kwargs)
+    elif input_data_type == InputDataType.INQUIRY:
+        return inquiry_reshaper(*args, **kwargs)
+    else:
+        raise NotImplementedError(f"Unsupported input data type: {input_data_type}")
+
+
+def grouper(iterable, n, fillvalue=None):
+    args = [iter(iterable)] * n
+    return zip_longest(*args, fillvalue=fillvalue)
+
+
+def inquiry_reshaper(trial_target_info: list,
+                     timing_info: list,
+                     eeg_data: np.ndarray,
```

Comment:
It may be helpful for us to make a Data model which contains some of these repeated arguments that are intrinsically related (ex. data, fs)

Reply:
Ok - happy to include that here if you know what things should be grouped.
If it requires some thinking/design, perhaps this should be a follow-up ticket?

Reply:
Here's a ticket about it for now - let me know if you prefer to address it in this PR - https://www.pivotaltracker.com/story/show/178294216